server/NucleiClassification/NucleiClassification.py

```diff
@@ -0,0 +1,181 @@
+import os
+import sys
+import json
+import colorsys
+
+import numpy as np
+import pandas as pd
+from sklearn.externals import joblib
+import dask.dataframe as dd
+
+from ctk_cli import CLIArgumentParser
+
+import logging
+logging.basicConfig(level=logging.CRITICAL)
+
+sys.path.append(os.path.normpath(os.path.join(os.path.dirname(__file__), '..')))
+from cli_common import utils as cli_utils # noqa
+
+
+def gen_distinct_rgb_colors(n, seed=None):
+    """
+    Generates N visually distinct RGB colors
```

Comment:
For what it's worth, the hue of HSV isn't perceptually uniform, which means that generated colors will "clump" visually. Since doing it right is a pain, hopefully this isn't an issue.